config/auto-aux/cpp.c

```diff
@@ -0,0 +1,6 @@
+/* This file should preprocess to the string "ok", modulo whitespace */
```

Comment:
That's what this check is for. `configure` tries running this through `cc -E -P`, to see if it produces `ok`.

Removing `#line` and `#file` with `-P` isn't specified by POSIX, but seems to be widely available. I've tested gcc and clang, and a quick look at docs for sun's cc and AIX cc say it exists (but I haven't run them). MSVC has this option under the name `/EP`, apparently.